### PR TITLE
feat(theme): forms

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
Floating labels on forms are pretty old UI pattern

**What is the chosen solution to this problem?**
Refreshing our forms style

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_forms.switch.scss [hunk 1]
```diff
@@ -6,91 +6,70 @@ $switch-track-shadow-disabled: inset 2px 2px 7px 0 rgba(0, 0, 0, 0.2);
...
-			border-radius: 13rem;
+		+ *::before,
+		+ *::after {
+			content: '';
+			position: absolute;
+			top: 0;
+			left: 0;
+			border: none;
+			transition: all 250ms ease-in-out; // sass-lint:disable-line declaration-property-value-disallowed-list
+			border-radius: 9999px;
```
Comment (tlnd-mhuchet) [inline packages/theme/src/theme/_forms.switch.scss:31]: It's Over 9000!

**Comment by tlnd-mhuchet** [comment]
Little glitch here
CleanShot 2021-08-31 at 16 02 57@2x

**Comment by tlnd-mhuchet** [comment]
Don't know if the mess with buttons is wanted 😅 
CleanShot 2021-08-31 at 16 03 38@2x

**Comment by tlnd-mhuchet** [comment]
Input for time is not large enough by default to display the placeholder, and the info button is quite close to the input

CleanShot 2021-08-31 at 16 05 25@2x

Btw if I try to choose a time, it is invalid 😅 

CleanShot 2021-08-31 at 16 07 02@2x

**Comment by tlnd-mhuchet** [comment]
The `=` is misplaced when there is no error
CleanShot 2021-08-31 at 16 09 56@2x

CleanShot 2021-08-31 at 16 11 01@2x

**Comment by tlnd-mhuchet** [comment]
Values does not seems to be vertically centered in tags

CleanShot 2021-08-31 at 16 14 33@2x

**Comment by tlnd-mhuchet** [comment]
Picto not visible after click

CleanShot 2021-08-31 at 16 18 11@2x

CleanShot 2021-08-31 at 16 19 20@2x

**Comment by tlnd-mhuchet** [comment]
Maybe picto and text could be aligned here

CleanShot 2021-08-31 at 16 23 12@2x

File: packages/theme/src/theme/_forms.switch.scss (hunk 1 above)
Comment (frassinier) [reply packages/theme/src/theme/_forms.switch.scss:32]: Under control

**Review by tlnd-mhuchet — APPROVED** [review]

**Comment by akaseka** [comment]
Not enough space for placeholder, info button to close to the input
photo_2021-09-09_17-58-17

**Comment by akaseka** [comment]
An issue with arrow icons
photo_2021-09-09_18-02-41

**Comment by akaseka** [comment]
The arrow icon isn't placed right on the Data range and Data time range
photo_2021-09-15_12-58-44

**Comment by akaseka** [comment]
The label is too close to the toggle button
photo_2021-09-15_12-59-58

**Comment by frassinier** [comment]
@akaseka and @tlnd-mhuchet, thank you for the errors you've spotted!
This pull request can not pretend to fix everything but I  did my best to fix what I can with a thumb up! 
And I've put a confused emoji where it is not related to theses changes! 🙌🏻

**Review by akaseka — APPROVED** [review]

**Pull request merged**